### DISCUSSION TITLE
Styling of Universal Calculator

### DISCUSSION THREAD

**Discussion by Satish-Mutyala** [Satish-Mutyala/T24S2T1EXTG12#4] [General]
Hi Team, are you happy with the Universal Calculator styling?
Please see the screenshot below. If we are ok with it, it would be great if we can maintain the same theme across all calculators.
![image](https://github.com/user-attachments/assets/8f4aed00-cbd8-49d0-9f9c-897a84a12289)

**Comment by WeiZhou-dev** [comment]
Hi Satish, I am happy with the layout design. Thanks for your hard work.

**Comment by JeongseopYun** [comment]
Hello Satish, I love the layout. Thank you for your help.
